src/components/toggle-button/toggle-button.stories.js

```diff
@@ -0,0 +1,172 @@
+import { html } from 'lit-html';
+import '.';
+import { action } from '@storybook/addon-actions';
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { docsIntro, TYPES, CATEGORIES } from '../../../.storybook/utils.js';
+import { ICON_PLACEMENT } from './enums';
+import { argsShared, argTypesShared } from '../button/config';
+
+export default {
+  title: 'custom-elements/vl-toggle-button',
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'toggle-button',
+          intro: 'The toggle button is a button that can have an active and inactive state.',
+        }),
+      },
+    },
+  },
+  args: {
+    ...argsShared,
+    icon: undefined,
```

Comment:
Ik zou zeker ook een story voorzien `With Icon`, net zoals we met de button hebben gedaan.